_layouts/contractor.html

```diff
@@ -43,3 +43,4 @@ <h3>Languages</h3>
     </section>
   </div>
 </div>
+</div>
```

Comment:
Please indent the rest of the code, leave the blank line following the front matter, and fix your editor to include newline at EOF.